### DISCUSSION TITLE
Strapi with Next-Auth

### DISCUSSION THREAD

**Discussion by erhankaradeniz** [nextauthjs/next-auth#1334] [Help]
Hi All,

I've got the login working with strapi and am able to login users. However Next-Auth creates its own tables in the database such as user, accounts etc. (it's documented in the Models section I know).

I was wondering if it's possible to make use of the strapi tables so users are manageable within strapi. Currently there is no way for me to see if a user is logged in with access to specific routes). 

How should I tackle this issue?

**Comment by vitali-duckma** [comment]
Hi, how you managed that'?
